dotcom-rendering/src/web/components/PinnedPostLiveness.importable.tsx

```diff
@@ -25,9 +27,41 @@ export const PinnedPostLiveness = ({}) => {
 		pinnedPost && pinnedPost.scrollHeight <= pinnedPost.clientHeight;
 	if (contentFitsContainer) hideShowMore();
 
+	if (pinnedPost) {
+		submitComponentEvent({
+			component: {
+				componentType: 'LIVE_BLOG_PINNED_POST',
+				id: pinnedPost?.id,
+			},
+			action: 'INSERT',
+		});
+	}
```

Comment:
I wanted to emit the `INSERT` event only when pinned post is loaded in the page. Meaning we only want to emit 1 event at the beginning. I'm not sure if it's right to just emit the event like this, and I think maybe, we need to create a isLoaded state or something like that to make sure we only emit the `insert` event when the page loads. 

Right now there's no state in this component `PinnedPostLiveness` so from what I see, there should not be any re-rendering. 

What do you think is the best way to tackle this?